src/test/suite/format/format.test.ts

```diff
@@ -73,6 +86,18 @@ suite('Formatter', () => {
             );
             const formatter = new FormatProvider();
 
+            // Set extension's option
+            let prevValue = null;
+            if (formatTest.extensionOption) {
+                prevValue = Global.getConfig<
```

Comment:
Do we need save previous config value if we test it? Maybe set non-default value to test and then remove it from configuration and VSCode will set it to default value?

Reply:
Sorry for the delay. I understand your concern here. See #237 for a similar concern and #239 for a solution. Let me work on that PR so we can unblock this one :)

Reply:
@kyklish try now :)